x-pack/plugins/stack_connectors/server/connector_types/openai/openai.ts

```diff
@@ -90,9 +120,21 @@ export class OpenAIConnector extends SubActionConnector<Config, Secrets> {
       method: 'invokeStream',
       schema: InvokeAIActionParamsSchema,
     });
+
+    this.registerSubAction({
+      name: SUB_ACTION.INVOKE_ASYNC_ITERATOR,
+      method: 'invokeAsyncIterator',
+      schema: InvokeAIActionParamsSchema,
+    });
   }
 
   protected getResponseErrorMessage(error: AxiosError<{ error?: { message?: string } }>): string {
+    if (error.message === '404 Unrecognized request argument supplied: functions') {
```

Comment:
ouch! so specific!  But I guess from context, it's from an older version, so seems ok.  Would there be headers or such we could check?

Reply:
I just went to check and I am unable to produce this error anymore... I would previously get it when using the `2023-03-15-preview` version. Not sure if they updated the API or what... unsure if I should leave this or not

Reply:
Since it's so specific, and you're not seeing it, I guess we can get rid of it.  If you're not sure, leaving it in is seems safe, but maybe add a comment regarding it being an "early release" thing for the lib and can eventually be deleted ...